src/curated-corpus/components/ProspectFilters/ProspectFilters.tsx

```diff
@@ -5,9 +5,30 @@ import {
   Grid,
   Switch,
   TextField,
+  Typography,
 } from '@mui/material';
+import { Prospect } from '../../../api/generatedTypes';
+import { DropDownFilter } from '../DropDownFilter/DropDownFilter';
+// import { ProspectDropDownFilter } from '../ProspectDropDownFilter/ProspectDropDownFilter';
```

Comment:
Forgotten import?

Reply:
@nina-py this seems outdated, I removed it in a recent commit!